backend/src/main/java/io/diveni/backend/config/CorsConfig.java

```diff
@@ -9,11 +11,14 @@
 @Configuration
 public class CorsConfig {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(CorsConfig.class);
+
   @Value("${SERVER_URL:#{null}}")
-  private String SERVER_URL;
+  private static String SERVER_URL;
```

Comment:
Injecting to a static variable does not work.
A workaround can be found [here](https://www.baeldung.com/spring-inject-static-field).

Reply:
Ups forgot to revert. Thanks.